app/models/contestant_project.rb

```diff
@@ -1,3 +1,5 @@
 class ContestantProject < ApplicationRecord
-
+  belongs_to :contestant
+  belongs_to :project
+  has_many :projects
```

Comment:
This is incorrect; a joined record will not `have_many` of one model and not the other. 
Another way to understand what a join table does would be to think of one row at a time - a row of the `contestant_projects` table has an ID (its primary key), a contestant_id, and a project_id. If it `has_many` of another field, we may need another join table to connect them further.... But we don't, because it connects itself to the Contestant and Project models with one foreign key listed here.